index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio | Edwin Gichuhi</title>
+    </head>
+
+    <body>
+        <nav class="toolbar">
+            <a href="#">John Doe</a>
+            <img src="/images/Icon-Menu.png" alt="menu">
+        </nav>
+        <section class="headline">
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <h3>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</h3>
+            <section class="socials">
+                <img src="/images/Icon-GitHub.png" alt="Github">
+                <img src="/images/Icon-Linkedin.png" alt="Github">
+                <img src="/images/Vector.png" alt="Github">
+                <img src="/images/Icon-Twitter.png" alt="Github">
+                <img src="/images/Icon-Meta.png" alt="Github">
+            </section>
+            <img src="/images/after-socials.png" alt="after-socials-img" class="after-social-img">
```

Comment:
:red_circle:  I think there is also something wrong with this image path. Please check the path of this image.

Reply:
Fixed.